pkg/detectors/digitaloceanv2/digitaloceanv2.go

```diff
@@ -21,13 +21,13 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(`\b(dop\_v1\_[a-f0-9]{64})\b`)
+	keyPat = regexp.MustCompile(`\b(dop\_v1\_[a-f0-9]{64} | doo\_v1\_[a-f0-9]{64} | dor\_v1\_[a-f0-9]{64})\b`)
```

Comment:
suggestion: Since the only differences in these regex are the first 3 characters we can just or that portion. The rest will remain the same. Maybe something like: `\b(?:(dop|doo|dor)_v1_[a-f0-9]{64})\b`. Additionally `_` does not need to be escaped so we can omit that.

Reply:
I modified the regex based on the suggestion.
Thank you.